conf/app.routes

```diff
@@ -137,14 +135,20 @@ PUT         /assessor/:assessorId/allocations                            control
 POST        /events/save                                                 controllers.EventsController.saveAssessmentEvents()
 POST        /events/create                                               controllers.EventsController.createEvent()
 GET         /events/:eventId/allocations                                 controllers.EventsController.getAssessorAllocations(eventId: String)
-GET         /events/:eventId/sessions/:sessionId/candidateAllocations    controllers.EventsController.getCandidateAllocations(eventId: String, sessionId: String)
-POST        /events/:eventId/sessions/:sessionId/remove-candidates       controllers.EventsController.removeCandidateAllocations(eventId: String, sessionId: String)
+
+# candidate allocations
+GET         /candidate-allocations/find-applications/location/:location            controllers.CandidateAllocationController.findCandidatesEligibleForEventAllocation(location: String)
+POST        /candidate-allocations/find-allocated                                  controllers.CandidateAllocationController.findAllocatedApplications()
+GET         /candidate-allocations/events/:eventId/sessions/:sessionId             controllers.CandidateAllocationController.getCandidateAllocations(eventId: String, sessionId: String)
+POST        /candidate-allocations/remove/events/:eventId/sessions/:sessionId      controllers.CandidateAllocationController.removeCandidateAllocations(eventId: String, sessionId: String)
```

Comment:
List of applicationIds are passed as body into this request.  Body is not supported in DELETE request.